pkg/hostagent/mount.go

```diff
@@ -68,6 +68,9 @@ func (a *HostAgent) setupMount(m limayaml.Mount) (*mount, error) {
 		Readonly:            !(*m.Writable),
 		SSHFSAdditionalArgs: []string{"-o", sshfsOptions},
 	}
+	if runtime.GOOS == "windows" {
```

Comment:
Could you add a code comment to the explain the reason

Reply:
Description added in code and in the commit message.